contents/projectManagement/githubActions.md

```diff
@@ -0,0 +1,155 @@
+![](images/editor.png)
+
+### 2. Reusable Actions
+Actions are small, reusable units in a workflow. When setting up a workflow, we can reuse actions written by the community. These actions are available at the marketplace.
+
+
+![](images/marketplace.png)
+
+
+
+### 3. Cross OS Support
+GitHub Actions gives you the option to [run workflows](https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources) on Ubuntu, Windows and macOS.
+
+### 4. Docker Support
+GitHub Actions provides the ability to run docker images for projects that use a dockerized environment.
+
+### 5. Powerful
+Integration with the GitHub ecosystem allows GitHub Actions to do more than just build/test/deploy. You can automate everything related to your GitHub project, including bug triaging, labeling, automated dependency updating etc.
+Workflows can be triggered by events that happen on GitHub and will have access to the context of the issue/PR/commit that triggered it. 
+
+### 6. Affordable
+Like many of its competitors, GitHub Actions provides **free unlimited runner time for public repositories**. 
+
+For private repositories, the free plan offers 2000 minutes of runner time per month and a pay-per-minute plan for additional minutes.
+
+---
+
+## How does GitHub Actions Work
+<!--This is a simple high-level overview of the tool to give the reader some concrete sense of X (as opposed to limiting to an entirely abstract description). It's useful to give concrete examples such as code examples. Do not try to 'teach' how to use the X (assuming your in the style of a tutorial. If the tool is worth learning, there must be good tutorials about it already.-->
+### Workflow
+A workflow describes a sequence of jobs to be run when a trigger event occurs. 
+Each job consists of a series of steps, which may be actions or shell commands. 
+Each workflow is defined as a `.yml` file in the `.github/workflows` directory of a repository.
+
+### Action
+An action is a piece of code that performs an atomic task. 
+For example, [checking out a repository](https://github.com/actions/checkout), or [caching dependencies](https://github.com/actions/cache).
+
+### Triggers
+Workflows can be triggered by events on the GitHub repository, ranging from opening pull requests or issues, making commits to scheduled events, or even from an external event by calling GitHub's REST API endpoint.
+
+---
+
+## How is GitHub Actions Being Used
+
+**Simple CI [workflow](https://github.com/actions/starter-workflows/blob/master/ci/node.js.yml) for a Node.js app**
+1. Trigger when a commit/pull request is made to the master branch
+2. Install dependencies
+3. Build
+4. Run tests with multiple versions of Node on Ubuntu
```

Comment:
It's better to use `1. ` for all items. Markdown generates the correct number for each item.